files/zh-cn/web/api/blob/arraybuffer/index.md

```diff
@@ -1,37 +1,33 @@
 ---
-title: Blob.arrayBuffer()
+title: Blob: arrayBuffer() 方法
```

Comment:
The problem why the test is failing is: the value of the metadata could not contain `:` without quotation marks wrapping. And for zh-CN, please use _full-width colon_（`：`）.

```suggestion
title: Blob：arrayBuffer() 方法
```